sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1066,6 +1102,7 @@ void ExtrinsicReflectorBasedCalibrator::trackMatches(
 
   active_tracks_.erase(timed_out_end, active_tracks_.end());
 
+  bool new_track = false;
```

Comment:
Any reason for bringing this outside with a value that does not seem used?

Reply:
It's a mistake! Thanks!
Commit https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f